resources/eventing/README.md

```diff
@@ -17,8 +22,8 @@ You can install this Helm chart using either Helm or Kyma CLI. In both cases, th
 
 ```bash
 $ cat << EOF > helm-values.yaml
-event-publisher-proxy:
-  upstreamAuthentication:
+eventing:
+  authentication:
     oauthClientId: "$bebOauthClientId"
     oauthClientSecret: "$bebOauthClientSecret"
     oauthTokenEndpoint: "$bebOauthTokenEndpoint"
```

Comment:
- top key should be authentication, not eventing.
- domain name is missing

```suggestion
global:
  domainName: "$domainName"
authentication:
  oauthClientId: "$bebOauthClientId"
  oauthClientSecret: "$bebOauthClientSecret"
  oauthTokenEndpoint: "$bebOauthTokenEndpoint"
```

Reply:
Changed

Reply:
Changed